browser/css/jsdialogs.css

```diff
@@ -1530,3 +1531,7 @@ div [id^='TemplateDialog'] {
 #CharacterPropertiesDialog {
 	min-width: 656px;
 }
+
```

Comment:
could you add `/* Spellchecking dialog */` comment to mark where we can find this?

Reply:
done